src/connector.ts

```diff
@@ -16,7 +16,7 @@ import { disabledPrettyPrinter } from './printer';
 import { keywordize } from './util/string';
 import { initializeDebugger } from './debugger/calva-debug';
 import * as outputWindow from './results-output/results-doc';
-import evaluate from './evaluate';
+import { evaluateInOutputWindow } from './evaluate';
```

Comment:
I prefer `import * as .. from` and thus to see from where something is called. We don't need to fix this now, just noting it for future discussions about code style.

Reply:
You can still use that, this just prevents you from calling a named export from a default export. https://github.com/import-js/eslint-plugin-import/blob/02ccbc1a30098822b7a6e7d84d8594b387495361/docs/rules/no-named-as-default-member.md